src/System.IO.Pipelines/ref/System.IO.Pipelines.csproj

```diff
@@ -12,7 +12,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\System.Buffers\ref\System.Buffers.csproj" />
-    <ProjectReference Include="..\..\System.Memory\ref\System.Memory.csproj" />
     <ProjectReference Include="..\..\System.Threading.Tasks.Extensions\ref\System.Threading.Tasks.Extensions.csproj" />
+    <Reference Include="System.Memory" />
```

Comment:
If there are any thing other then netstandard configuration, like netcoreapp, this will be a race condition. Can you condition this ItemGroup to ensure that doesn't happen.